packages/design-system/src/components/N8nCheckbox/__tests__/__snapshots__/Checkbox.spec.ts.snap

```diff
@@ -10,7 +10,6 @@ exports[`components > N8nCheckbox > should render with both child and label 1`]
       class="el-checkbox__input"
     >
       <input
-        aria-hidden="false"
```

Comment:
can we keep update the components to ensure that we don't lose these accessibility attributes, instead of removing them from the test snapshot 🙏🏽

Reply:
I don't think this is in our control since this is the attribute added by the `element-plus` library. Looking at [the docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden), `false` for this attribute doesn't do much (will not even re-expose the element to assistive technology if any of its parents are hidden).